static/app/components/organizations/pageFilters/persistence.tsx

```diff
@@ -34,7 +34,8 @@ type StoredObject = {
  */
 export function setPageFiltersStorage(
   orgSlug: string,
-  updateFilters: Set<PinnedPageFilter>
+  updateFilters: Set<PinnedPageFilter>,
+  storageNamespace?: string
```

Comment:
```suggestion
  storageNamespace: string = '',
```

m: This way we can remove the need for the optional chain inside the template string below. Ditto for `getPageFilterStorage` defn

Reply:
Are you referring to this? https://github.com/getsentry/sentry/blob/cbff28d0c197b92c9e8f4bfbbeb7138481b8b4a1/static/app/components/organizations/pageFilters/persistence.tsx#L104

I'll still have to conditionally check if it's an empty string, since we need to include a `:` in the prefix, but can't add it if it's empty 🤔

Reply:
i may have not seen the colon...

![the-simpsons-homer-simpson](https://github.com/getsentry/sentry/assets/18689448/c667e3fa-2d39-4caf-9cc1-b5ee26a1d1c9)

maybe we turn the ternary into:

```js
storageNamespace.length > 0 ? `${storageNamespace}:${orgSlug}` : orgSlug
```

which feels a little more readable to me.

Reply:
Agreed on the readability! Updated in both spots